exporters/ibmmq/exporter.yml

```diff
@@ -32,3 +32,5 @@ exporter_guid: f27b1ade-938f-4594-a752-7d4bb1daabf1
 license_guid: 2b9425a0-3b30-4122-bd09-4b375c328c75
 # This GUID should be generated and be unique across all exporters in the repository
 config_guid: 8c224fdc-70ce-4f07-b3e6-2c97b6596c41
+
+# Force execution for testing purposes.
```

Comment:
I don't understand this comment here.

Reply:
I needed an `exporter.yml` change so the pipeline triggers and I was able to test that the changes in the pipeline were correct. Removed the change in [#a6c8a26](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/172/commits/a6c8a2694cb3108f405f38a4f1631daff2633305)